test/parallel/test-tls-multi-key.js

```diff
@@ -30,12 +31,12 @@ const fs = require('fs');
 
 const options = {
   key: [
-    fs.readFileSync(`${common.fixturesDir}/keys/ec-key.pem`),
-    fs.readFileSync(`${common.fixturesDir}/keys/agent1-key.pem`),
+    fs.readFileSync(fixtures.path('/keys/ec-key.pem')),
```

Comment:
This could be using method `fixtures.readKey` to get the keys.

Reply:
Does it make a difference that fs.readFileSync() is platform specific (though only if the path is a directory I see...so not an issue). I've always been taught to keep it as simple as possible...so I assumed using path.join() is just a bit clearer? But live and learn! :)

Reply:
The readFileSync(fixtures(keys...)) pattern is so common in the tests that the `fixtures.readKey` wrapper was created to handle it. Take a look at [`fixtures.readKey()`](https://github.com/nodejs/node/tree/master/test/common#fixturesreadkeyarg-enc)!

Something like `fixtures.readKey('ec-key.pem')`.